src/actions/tweet.actions.js

```diff
@@ -28,12 +28,13 @@ export const tweet = (data, history) => dispatch => {
     }
     return axios
         .post(
-            url, 
-            data, 
+            url,
+            data,
             {headers: headers},
           )
         .then(response => {
-            console.log('hfhghgcvbxgcbcvv', response)
+            // console.log('hfhghgcvbxgcbcvv', response)
+            console.log(response);
```

Comment:
You could do away with console here

Reply:
This was part of your task implementation @susanwere , I forgot to ask you what they were doing. I will remove them then